sys-apps/tas/tas-1.5.2.180622.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eapi7-ver linux-info systemd
+
+CONFIG_CHECK="~IPMI_DEVICE_INTERFACE ~IPMI_HANDLER ~IPMI_SI"
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="${PN^^}"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="Supermicro Thin-Agent Service for monitoring through the BMC/IPMI"
+HOMEPAGE="https://www.supermicro.com"
+SRC_URI="${MY_PN}_${MY_PV}_build.${MY_DATE}.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="BSD supermicro"
+SLOT="0"
+
+RDEPEND="net-misc/networkmanager
+	sys-apps/ethtool
+	sys-apps/net-tools
+	sys-apps/smartmontools
+	sys-block/storcli
+	sys-devel/bc
+	sys-fs/lsscsi
+	sys-fs/mdadm"
+DEPEND="app-arch/unzip"
+
+RESTRICT="bindist fetch mirror strip"
```

Comment:
Why strip?

Reply:
Isn't this necessary, as the license states `THE LICENSEE MAY NOT MODIFY, ADAPT, TRANSLATE,
  RENT, LEASE, LOAN, RESELL FOR PROFIT, DISTRIBUTE, OR OTHERWISE ASSIGN OR
  TRANSFER THE SOFTWARE, OR CREATE DERIVATIVE WORKS BASED UPON THE SOFTWARE
  OR ANY PART THEREOF, EXCEPT AS EXPRESSLY AGREED TO IN WRITING BY
  SUPERMICRO.`?

Reply:
I think we've already established that stripping doesn't really count here.